pages/landing-page.js

```diff
@@ -0,0 +1,437 @@
+// @ts-check
+// eslint-disable-next-line no-unused-vars
+import { ReactElement, useEffect, useState } from 'react'; // ReactElement is only imported for typechecking
+import Footer from '../components/Footer';
+import Head from 'next/head';
+import Image from 'next/image';
+import PropTypes from 'prop-types';
+import { useRouter } from 'next/router';
+
+// import { BsSearch } from 'react-icons/bs';
```

Comment:
Hey, I'm curious why you decided to put this in, then comment it out? I vaguely remember it being in the navbar?

Reply:
That element has been commented out, in addition to a few other elements in the navbar which here is a copy of the original code with redactions. Depending on the decisions made with the navbar here, I may modify the original to make it more suitable to these broader cases where elements may want to be added/remove per page.